great_expectations/validator/validator.py

```diff
@@ -639,8 +639,6 @@ def _update_metric_value_kwargs_for_success_keys(
                 parameter_builder.metric_value_kwargs or {}
             )
 
-            key: str
-            value: Any
```

Comment:
So @alexsherstinsky tends to add these as a way of annotating the values in a comprehension - I would probably keep them but I'm okay either way.